selfdrive/car/gm/interface.py

```diff
@@ -135,6 +146,46 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, disa
       ret.lateralTuning.pid.kf = 0.000045
       tire_stiffness_factor = 1.0
 
+    elif candidate == CAR.SILVERADO:
+      ret.minEnableSpeed = -1.
+      ret.mass = 2400. + STD_CARGO_KG
+      ret.wheelbase = 3.745
+      ret.steerRatio = 16.3
+      ret.centerToFront = ret.wheelbase * .49
+      ret.networkLocation = NetworkLocation.fwdCamera # Uses Cam Harness
+      ret.radarOffCan = True # No Radar
+      ret.openpilotLongitudinalControl = False # Stock ACC
+      ret.pcmCruise = True # CC is on
+      # Tune
+      ret.steerRateCost = .4
+      ret.steerActuatorDelay = 0.11
+      ret.lateralTuning.pid.kpBP = [[10., 41.0], [10., 41.0]]
+      ret.lateralTuning.pid.kpV = [0.13, 0.23]
+
+    elif candidate == CAR.BOLT_EUV:
+      ret.transmissionType = TransmissionType.direct # EV (or hybrid)
+      ret.minEnableSpeed = -1
+      ret.mass = 1669. + STD_CARGO_KG
+      ret.wheelbase = 2.675
+      ret.steerRatio = 16.8 # Not measured
+      ret.centerToFront = ret.wheelbase * 0.4
+      ret.networkLocation = NetworkLocation.fwdCamera # Uses Cam Harness
+      ret.radarOffCan = True # No Radar
+      ret.openpilotLongitudinalControl = False # Stock ACC
+      ret.pcmCruise = True # CC is on
+      # Tune
```

Comment:
FYI - Trilobyte is our EUV tester, and this may be an old tune. I discovered an error in the params, and I would like to see if the default tune might be acceptable, or if it needs adjustment (centerToFront was about 2x what it should have been)

Reply:
Do you know if the EUV has similar torque to the other GMs with the same units?